tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,14 +87,27 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+    }
+
     it should "show api build version" in {
+        wsk.cli(wskprops.overrides ++ Seq("property", "set"))
         val stdout = wsk.cli(wskprops.overrides ++ Seq("property", "get", "--apibuild")).stdout
         stdout should include regex ("""(?i)whisk API build\s+201.*""")
+        wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"))
```

Comment:
IMHO, each test should not assume there are any particular values in the .wskprops file.

On a fresh system, the file should be empty - but is set with the default values upon the first wsk CLI invocation.  Currently, most tests explicitly set the `--auth` and other properties on the command line so those test do no rely on what is in the .wskprops file.

If a test needs a particular value actually in the .wskprops file, the test should explicitly set that value and then run the test.  After the test, unsetting the value back to the default seems preferred.